specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/MitreCoverage.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Insights",
+    "description": "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider",
+    "version": "2022-01-01-preview",
+    "x-ms-code-generation-settings": {
+      "name": "SecurityInsightsClient"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/mitreCoverage": {
```

Comment:
>mitreCoverage [](http://example.com/codeflow?start=176&length=13)

Mitre seems to be a product or project name. Can you change the name of this resource to indicate what it contains or what it does?

Reply:
you are totally right "Mitre" is the name for a globally-accessible knowledge base of adversary tactics and techniques based on real-world observations on the security world.
that's why we called it "**MitreCoverage**" (and not just Mitre). Mitre coverage is responsible to return the MITRE - coverage of products within Sentinel. means - how much a customer is covered from Mitre Attacks.
i feel confident with the name. and we had an internal discussion, in which we agreed on that name. 
@mentat9